apps/src/code-studio/components/progress/StageLockDialog.jsx

```diff
@@ -0,0 +1,284 @@
+import React from 'react';
+import Radium from 'radium';
+import { connect } from 'react-redux';
+import BaseDialog from '@cdo/apps/templates/BaseDialog';
```

Comment:
Playing with the feature on adhoc...I actually don't think we need those extra instructions.  The dialog stays up until you hit "save" or "cancel" and you see the table cells changing.  So it becomes obvious that you need to hit Save.